react/features/base/tracks/functions.js

```diff
@@ -197,6 +258,19 @@ export function getTracksByMediaType(tracks, mediaType) {
     return tracks.filter(t => t.mediaType === mediaType);
 }
 
+/**
+ * Checks if the local video track in the given set of tracks is muted.
+ *
+ * @param {Track[]} tracks - List of all tracks.
+ * @returns {Track[]}
+ */
+export function isLocalVideoTrackMuted(tracks) {
+    const presenterTrack = getLocalTrack(tracks, MEDIA_TYPE.PRESENTER);
+
+    return presenterTrack ? isLocalTrackMuted(tracks, MEDIA_TYPE.PRESENTER)
```

Comment:
The suggested change doesn't work since isLocalTrackMuted returns true when the track is not present. So returns true even though video exists when not screensharing.

Reply:
Makes sense. Maybe add a comment please, since it isn't obvious at first glance.

Reply:
Sure, I had to add another check as well.